### PR TITLE
updated a few things 

### PR DESCRIPTION
**Pull request by ghost**
fixed the grammar a bit.

### CODE REVIEW

File: docs/admin_guide.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
 # Admin Bare Metal/VM Guide
 
 Below is a step-by-step guide of the process for creating your own /kbin instance from the moment a new VPS/VM is created or directly on bare-metal.  
-This is a preliminary outline that will help you launch an instance for your own needs.
+This is a preliminary outline that will help you launch an instance for your needs.
```
Comment (garrettw) [inline docs/admin_guide.md:4]: "own" is not a problem here

File: docs/admin_guide.md [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
-For Docker see: [Admin Deployment Guide](./docker_deployment_guide.md).
+For Docker, see: [Admin Deployment Guide](./docker_deployment_guide.md).
 
 > **Note**
 > /kbin is still in the early stages of development.
 
 If you would like to support the project, you can register using the following [affiliate link](https://hetzner.cloud/?ref=8tSPCw0qqIwl).
 
-This guide is aimed for Debian / Ubuntu distribution servers, but it could run on any modern Linux distro. This guide will however uses the `apt` commands.
+This guide is aimed for Debian / Ubuntu distribution servers, but it could run on any modern Linux distro. This guide will, however, will use the `apt` commands.
```
Comment (garrettw) [inline docs/admin_guide.md:13]: This is still bad; you repeated "will". Just make it "will, however, use"

File: docs/admin_guide.md [hunk 3]
```diff
@@ -243,15 +243,15 @@ sudo nano /etc/redis/redis.conf
 # Change no to systemd, considering Ubuntu is using systemd
 ```
 
-Save and exit (ctrl+x) the file.
+Save and exit (CTRL+x) the file.
```
Comment (garrettw) [inline docs/admin_guide.md:246]: If you consider "ctrl" to be wrong, then so is "CTRL". It's not an acronym, it's the word "Control", so use what most keyboards use: "Ctrl"

File: docs/admin_guide.md [hunk 4]
```diff
@@ -264,7 +264,7 @@ REDIS_DNS=redis://${REDIS_PASSWORD}@${REDIS_HOST}
 
 ### PostgreSQL (Database)
 
-Create new `kbin` database user, using the password, `{!SECRET!!KEY!-32_2-!}`, you generated earlier:
+Create a new `kbin` database user, using the password, `{!SECRET!!KEY!-32_2-!}`, you generated earlier:
```
Comment (garrettw) [inline docs/admin_guide.md:267]: This fix is correct, but the sentence is still awkward and needs more work.
I would suggest something like this:
Create a new `kbin` database user, using the password you generated earlier (`{!SECRET!!KEY!-32_2-!}`):

File: docs/admin_guide.md [hunk 5]
```diff
@@ -771,17 +771,17 @@ php bin/console kbin:user:admin <username>
 php bin/console kbin:ap:keys:update
 ```
 
-Next, log in and create a magazine named "random" to which unclassified content from the fediverse will flow.
+Next, log in and create a magazine named “random” to which unclassified content from the Fediverse will flow.
```
Comment (garrettw) [inline docs/admin_guide.md:774]: I'm not sure that adding smart quotes is useful or necessary.

File: docs/docker_deployment_guide.md [hunk 6]
```diff
@@ -115,23 +115,23 @@ docker compose exec php bin/console kbin:user:admin <username>
 docker compose exec php bin/console kbin:ap:keys:update
 ```
 
-Next, log in and create a magazine named "random" to which unclassified content from the fediverse will flow.
+Next, log in and create a magazine named “random” to which unclassified content from the Fediverse will flow.
 
 ### Add auxiliary containers to `docker-compose.yml`
 
-Add any auxiliary container as you want. For example, add a Nginx container as reverse proxy to provide HTTPS encryption.
+Add any auxiliary container as you want. For example, add a Nginx container as a reverse proxy to provide HTTPS encryption.
```
Comment (garrettw) [inline docs/docker_deployment_guide.md:122]: "Nginx" is a strange word when it comes to adding an indefinite article in front of it. One might think "a" would be correct since the word starts with "n", but... when I read through this sentence out loud, I naturally correct "a" to "an" because "Nginx" is pronounced "engine x" and not "n-jinx".

File: docs/docker_deployment_guide.md [hunk 7]
```diff
@@ -115,23 +115,23 @@ docker compose exec php bin/console kbin:user:admin <username>
...
-Uploaded media files (e.g. photos uploaded by users) will be stored on the host directory `storage/media`. They will be served by the Caddy web server in the `www` container as static files.
+Uploaded media files (e.g., photos uploaded by users) will be stored on the host directory `storage/media`. They will be served by the Caddy web server in the `www` container as static files.
 
-Make sure `KBIN_STORAGE_URL` in your `.env` configuration file is set to be `https://yourdomain.tld/media` (assuming you setup Nginx with SSL certificate by now).
+Make sure `KBIN_STORAGE_URL` in your `.env` configuration file is set to be `https://yourdomain.tld/media` (assuming you set up Nginx with SSL certificate by now).
 
 You can also serve those media files on another server by mirroring the files at `storage/media` and changing `KBIN_STORAGE_URL` correspondingly.
 
-## Filesystem ACL support
+## File system ACL support
```
Comment (garrettw) [inline docs/docker_deployment_guide.md:132]: "Filesystem" as one word in English is fine. It's frequently used that way.

File: docs/docker_deployment_guide.md [hunk 8]
```diff
@@ -115,23 +115,23 @@ docker compose exec php bin/console kbin:user:admin <username>
...
 
-The filesystem ACL is disabled by default, in the `kbin` image. You can set the environment variable `ENABLE_ACL=1` to enable it. Remember that not all filesystems support ACL. This will cause an error if you enable filesystem ACL for such filesystems.
+The file system ACL is disabled by default, in the `kbin` image. You can set the environment variable `ENABLE_ACL=1` to enable it. Remember that not all file systems support ACL. This will cause an error if you enable file system ACL for such file systems.
```
Comment (garrettw) [inline docs/docker_deployment_guide.md:134]: Undo this change too

File: docs/oauth2_guide.md [hunk 9]
```diff
@@ -21,7 +21,7 @@
 ## Available Scopes
 ### Scope tree
 1. `read` - Allows retrieval of threads from the user's subscribed magazines/domains and viewing the user's favorited entries.
-2. `write` - Provides all of the following nested scopes
+2. `write` - Provides all the following nested scopes
```
Comment (garrettw) [inline docs/oauth2_guide.md:24]: "of" is actually more correct here.

File: docs/oauth2_guide.md [hunk 10]
```diff
@@ -30,7 +30,7 @@
     * `post:edit`
     * `post_comment:create`
     * `post_comment:edit`
-3. `delete` - Provides all of the following nested scopes, for deleting the current user's content
+3. `delete` - Provides all the following nested scopes, for deleting the current user's content
```
Comment (garrettw) [inline docs/oauth2_guide.md:33]: Same here -- "of" does belong

File: docs/oauth2_guide.md [hunk 11]
```diff
@@ -153,7 +153,7 @@
             * `moderate:magazine:reports:read`
                 * Allows the client to read reports about content from magazines the user moderates
             * `moderate:magazine:reports:action`
-                * Allows the client to take action on reports, either accepting or rejecting them
+                * Allows the client to act on reports, either accepting or rejecting them
```
Comment (garrettw) [inline docs/oauth2_guide.md:156]: I don't think either version here is better than the other

File: docs/user_guide.md [hunk 12]
```diff
@@ -8,42 +8,42 @@ The initiative aims to promote a free and open internet.
 ## Introduction
 
 The platform is divided into thematic categories called magazines. By default, any user can create their own magazine
-and automatically become its owner. Then they receive a number of administrative tools that will help them personalize
+and automatically become its owner. Then they receive many administrative tools that will help them personalize
```
Comment (garrettw) [inline docs/user_guide.md:11]: This is not necessarily the same meaning. "Many" implies that the number of tools is large, while "a number of" generally carries the connotation of "maybe many, but at least several"

File: docs/user_guide.md [hunk 13]
```diff
@@ -8,42 +8,42 @@ The initiative aims to promote a free and open internet.
...
-Platform is equally suitable for a small personal instance for friends and family, a school or university community,
+The platform is equally suitable for a small personal instance for friends and family, a school or university community,
 company platform or a general instance with thousands of active users.
 
 ## User guide
 
 ### Customization
 
-Everyone has the ability to customize the appearance to suit your preferences. In the sidebar, you'll find an options
+Everyone can customize the appearance to suit their preferences. In the sidebar, you'll find an option
```
Comment (garrettw) [inline docs/user_guide.md:28]: "options" is more correct here, as the button's purpose is to reveal "a variety of settings". If the button used text rather than just an icon, it would more likely say "Options" rather than "Option".

File: docs/user_guide.md [hunk 14]
```diff
@@ -8,42 +8,42 @@ The initiative aims to promote a free and open internet.
...
 enable automatic refreshing of posts and comments, activate infinite scroll, and enable automatic media previews.
 
 By using these options, you can completely transform the appearance of the platform to fit your personal style. Whether
 you prefer a minimalist design or a more colorful and lively look, you can easily make the changes that will make your
-experience on platform more enjoyable.
+experience on the platform more enjoyable.
 
 So don't be afraid to experiment with the various options available in the sidebar. You might be surprised at just how
-much you can change the appearance of the platform to suit your preferences.
+much, you can change the appearance of the platform to suit your preferences.
```
Comment (garrettw) [inline docs/user_guide.md:37]: This is 100% incorrect.

**Review by garrettw — CHANGES_REQUESTED** [review]
Overall a lot of improvements here, but it's definitely not ready to be merged.

File: docs/admin_guide.md (hunk 2 above)
Comment (aaronliu0130) [reply docs/admin_guide.md:13]: Hey, you can suggest changes on GitHub by specifying the triple-tick language to be "suggestion" and filling in your preferred version of the line like this!
```suggestion
This guide is aimed for Debian / Ubuntu distribution servers, but it could run on any modern Linux distro. This guide will, however, use the `apt` commands.
```
This way, they can easily implement your requests and you'll also get credited as the author of the commit.
